lib/smart_city/helpers.ex

```diff
@@ -46,9 +46,13 @@ defmodule SmartCity.Helpers do
   """
   @spec mime_type(file_type()) :: mime_type()
   def mime_type(file_type) do
-    case MIME.valid?(file_type) do
+    downcased_type = String.downcase(file_type)
+
+    downcased_type
+    |> MIME.valid?()
+    |> case do
       true -> file_type
-      false -> MIME.type(file_type)
+      false -> MIME.type(downcased_type)
```

Comment:
I think you're complicating this pipe a bit.

```elixir
case MIME.valid?(downcased_type) do
  true -> file_type
  false -> MIME.type(downcased_type)
end
```

Reply:
Also is there a reason we would return the uppercased type? Why not return the downcased_type or the converted type?

```elixir
case MIME.valid?(downcased_type) do
  true -> downcased_type
  false -> MIME.type(downcased_type)
end
```

Reply:
returning the original type was a mistake; result of incorrectly handling the situation where the input format was already a valid mime type.

i believe the pipe was the result of having started piping everything through before realizing i needed a handle on the intermediary value (the downcase) in order to validate it and then pass it along.